packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -179,28 +312,89 @@ class DatePickerInput extends React.Component {
     ]);
     const buttonText = intl.formatMessage({ id: 'Terra.datePicker.openCalendar' });
 
+    const dateMonthInput = (
+      <Input
+        {...additionalInputProps}
+        refCallback={(inputRef) => { this.monthInput = inputRef; }}
+        className={cx('date-input-month')}
+        type="text"
+        name={'terra-date-month-'.concat(name)}
+        value={this.state.month}
+        onChange={this.handleMonthChange}
+        placeholder={placeholderValues.month || 'MM'}
+        onFocus={this.handleOnFocus}
+        onBlur={this.handleOnBlur}
+        maxLength="2"
+        size="2"
+        pattern="\d*"
+        aria-label={intl.formatMessage({ id: 'Terra.datePicker.monthLabel' })}
+      />
+    );
+
+    const dateDayInput = (
+      <Input
+        {...additionalInputProps}
+        refCallback={(inputRef) => { this.dayInput = inputRef; }}
+        className={cx('date-input-day')}
+        type="text"
+        name={'terra-date-day-'.concat(name)}
+        value={this.state.day}
+        onChange={this.handleDayChange}
+        placeholder={placeholderValues.day || 'DD'}
+        onFocus={this.handleOnFocus}
+        onBlur={this.handleOnBlur}
+        maxLength="2"
+        size="2"
+        pattern="\d*"
+        aria-label={intl.formatMessage({ id: 'Terra.datePicker.today' })}
```

Comment:
Why should it read "today"?

Reply:
It needs a new translation string, this is just a temp to get things working until the PR is at the point where it is worth sending out for translations.